conf/variant/scatest-qemux86-64/local.conf.sample

```diff
@@ -12,3 +12,4 @@ PATCHRESOLVE = "noop"
 SCA_VERBOSE_OUTPUT = "1"
 
 INHERIT += "rm_work"
+INHERIT += "cve-check"
```

Comment:
That should not be enforce by the template IMO.
It can be selected through the configure script still - or ofc by the user's choice - but it shouldn't be the default

Reply:
the meta-sca way of doing this is to add `SCA_AVAILABLE_MODULES += "cvecheck"`  into local.conf

Reply:
but not in the template

Reply:
another way is to to run `buildntest` with `cvecheck` as an argument

Reply:
ok, sure thing, I removed the commit.